manual/development-process/access-an-environment-db.md

```diff
@@ -0,0 +1,102 @@
+#Accessing an environment's database
+
+Connecting to a database must be done through an AWS Bastion host, below are the steps detailing how to do this
+
+##1. Downloading and installing the Session Manager plugin
+First you need to download and install the AWS Session Manager plugin
+
+   * [Instructions for Mac (Step 1 to 3)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-macos)
+   * [Instructions for Ubuntu (Step 1 to 2)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-debian)
+   * [Instructions for Amazon Linux (Step 1 to 2)](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html#install-plugin-linux)
+
+You can check if it worked by running  `session-manager-plugin`, which should show you the message
+
+`The Session Manager plugin is installed successfully. Use the AWS CLI to start a session.`
+
+##2. Obtaining the Instance ID for the Bastion instance
+
+Next, you need to get the instance id for the bastion instance which can be found via command line or through the AWS console.
+
+###Via Command line (after you've updated your credentials)
+
+   Run this `aws ec2 describe-instances --profile integration` (or whatever the name of your integration profile is; if you are not sure or if it's failing, omit the `--profile integration` part before running and see if it works)
+
+   It will print out a JSON like this:
+
+       "Reservations": [
+           {
+               "Groups": [],
+               "Instances": [
+                   {
+                       "AmiLaunchIndex": x,
+                       "ImageId": "xxxxxxxx",
+                       "InstanceId": "xxxxxxxxx",
+       ...etc
+
+Copy the value of the `InstanceId`
+
+###Via Console
+
+Alternatively, you can also get it from the console:
+
+- In the search bar, type in "EC2" 
+- Under "Resources", click "Instances"
+- Find the "bastion-ec2-instance-intg" instance
+- Copy the "Instance ID"
+
+##3. Connecting via Session Manager
+Now you need to connect via the session manager by running the command
+
+   `ssm aws ssm start-session --target <the InstanceId you've copied> --profile integration`
```

Comment:
Is the `ssm` at the start of this command a typo? The AWS CLI commands normally start with `aws`, so `ssm aws ssm ...` looks a bit odd.

Reply:
Does `ssm ...` do something with the session manager plugin?